data/859/017/75/85901775.geojson

```diff
@@ -31,7 +31,7 @@
         "Floreasca"
     ],
     "name:eng_x_preferred":[
-        "Sala Floreasca"
+        "Floreasca"
```

Comment:
Let's remove this record from the PR. Looks like a legitimate place; this PR will only track deprecated / not current neighbourhood changes.

Reply:
There is no such neighbourhood - "Sala Floreasca", it is Floreasca. 
You prefer that renames and polygon adjustments be done in a separate PR ?